activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -182,6 +182,24 @@ def drop_current(environment = env)
         }
       end
 
+      def truncate_tables(configuration)
+        ActiveRecord::Base.connected_to(database: { truncation: configuration }) do
+          table_names = ActiveRecord::Base.connection.tables
+          internal_table_names = [
+            ActiveRecord::Base.schema_migrations_table_name,
+            ActiveRecord::Base.internal_metadata_table_name
+          ]
+
+          class_for_adapter(configuration["adapter"]).new(configuration).truncate_tables(*table_names.without(*internal_table_names))
+        end
+      end
+
+      def truncate_all(environment = env)
+        ActiveRecord::Base.configurations.configs_for(env_name: environment).each do |db_config|
```

Comment:
I'm quite not sure whether we should truncate replica databases? (We should pass `include_replicas: true` to the `configs_for` if we want to do it).